cleanlab/outlier.py

```diff
@@ -504,6 +504,8 @@ def _get_ood_predictions_scores(
                     f"params['adjusted_pred_probs'] = False. "
                 )
             else:
+                if isinstance(labels, list):
+                    labels = np.array(labels)
```

Comment:
You can instead use this function to convert to array: 
https://github.com/cleanlab/cleanlab/blob/master/cleanlab/internal/validation.py#L180
without type-checking the input. Will make it automatically work for pd.Series and list